app/controllers/help_controller.rb

```diff
@@ -40,19 +40,33 @@ def create
     end
 
     if @support_form.valid?
-      template_id = GOV_NOTIFY_CONFIG['help_email']['template_id']
+      if ENV['ZENDESK_API_ENDPOINT'].present?
+        UseCases::Administrator::CreateSupportTicket.new(
+          tickets_gateway: Gateways::ZendeskSupportTickets.new
+        ).execute(
+          requester: {
+            email: sender_email,
+            name: params[:support_form][:name] || current_user&.name,
+            organisation: sender_organisation_name
+          },
+          details: params[:support_form][:details]
+        )
+      else
```

Comment:
This else branch will lose all the notify code once the specs have been sorted out.